src/app/portfolio/helpers.ts

```diff
@@ -181,11 +181,18 @@ export const filterAndSortAssets = (
       .filter(Boolean)
       // Apply low balances filter if necessary
       .filter((asset) => {
-        return (
-          asset?.balanceUSD !== undefined &&
-          !(hideLowBalance && asset.balanceUSD <= 1)
-        );
+        return !(hideLowBalance && (asset?.balanceUSD || 0) <= 1);
       })
+      // Hide all asset without logo (mostly spam coins for EVM)
+      .filter((asset) => {
+        return asset?.logo !== "";
+      })
+      // Remove 0.0000 balances
+      // formatAmount return (0) in case of truncase 0.00000
+      .filter((asset) => {
+        return formatAmount(asset?.balanceMainUnit, 5) !== "0";
+      })
+
```

Comment:
Maybe more efficient to merge the 3 `filter()` calls into one?

Reply:
It's a tradeoff, I prefer to divide by 3 to have a more readable filter than a big filter that is hard to maintain.